public/js/calendar.js

```diff
@@ -1,69 +1,61 @@
+const commonCalendarOptions = {
+  plugins: ['interaction', 'timeGrid', 'dayGrid'],
+  views: {
+    timeGridOneDay: {
+      type: 'timeGrid',
+      duration: { days: 1 },
+      buttonText: 'nap',
+      slotLabelFormat: {
+        hour: 'numeric',
+        minute: '2-digit',
+        omitZeroMinute: false,
+      },
+      nowIndicator: true,
+    },
+  },
+  buttonText: {
+    today: 'ma',
+    month: 'hónap',
+    week: 'hét',
+  },
+  nowIndicator: true,
+  locale: 'hu',
+  selectable: true,
+  eventClick: (calEvent) =>
+    (location.href = `/groups/${calEvent.event.groupId}`),
+}
+
 function generateWebCalendar(data, calendarEl, room) {
   const calendar = new FullCalendar.Calendar(calendarEl, {
-    plugins: ['interaction', 'timeGrid', 'dayGrid'],
+    ...commonCalendarOptions,
     header: {
       left: 'prev,next today',
       center: 'title',
-      right: 'timeGridOneDay,timeGridWeek,dayGridMonth'
+      right: 'timeGridOneDay,timeGridWeek,dayGridMonth',
```

Comment:
Unnecessary comma.

Reply:
It's actually called trailing commas, and I got to like it these days. They make the diff simpler if you add new lines after. But we can talk about it and maybe change the linter accordingly.